examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/screens/paywall/PaywallScreen.kt

```diff
@@ -29,8 +31,40 @@ fun PaywallScreen(
                 Text(text = "Error: ${state.errorMessage}")
             }
             is PaywallScreenState.Loaded -> {
-                PaywallView(PaywallViewOptions.Builder().setOffering(state.offering).build())
+                PaywallView(
+                    PaywallViewOptions.Builder()
+                        .setOffering(state.offering)
+                        .setListener(viewModel)
+                        .build(),
+                )
+                state?.dialogText?.let {
+                    PurchaseAlertDialog(viewModel, it)
+                }
             }
         }
     }
 }
+
+@Composable
+private fun PurchaseAlertDialog(
```

Comment:
We might want to extract this to a different file... But for now, I think it's fine to keep here.